test-util/src/main/java/io/zeebe/test/util/Strings.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.test.util;
+
+import java.util.UUID;
+
+public class Strings {
+
+  /** @return a prefixed random UUID which can be used as a valid ID for a BPMN element */
+  public static String newRandomValidBpmnId() {
+    return "a" + UUID.randomUUID().toString();
```

Comment:
Why this prefix?

Reply:
Because a UUID can start with a number, and then it isn't a valid ID for a BPMN element (it will be rejected when parsing the XML) :(

Reply:
Fine for me was not sure why we need this